src/components/MetadataPanel/style.css

```diff
@@ -63,7 +63,7 @@
 
 .row {
     display: flex;
-    font-weight: 100;
+    font-weight: 400; 
```

Comment:
nittiest nit: stray space
```suggestion
    font-weight: 400;
```

Reply:
ty!